app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -154,7 +154,9 @@ def scan_job_create(entity)
   end
 
   def raw_scan_job_create(target_class, target_id = nil)
-    target_class, target_id = target_class.class.name, target_class.id if target_class.kind_of?(ContainerImage)
+    if target_class.kind_of?(ContainerImage)
+      target_class, target_id = target_class.class.name, target_class.id
```

Comment:
is `target_class` misnamed?  Should it really be `target` or `target_object`, because `target_class.id` seems to be non-sensical.